### PR TITLE
Allow html rendering of sponsor message

### PR DESCRIPTION
**Pull request by brianjp93**

### CODE REVIEW

**Comment by brianjp93** [comment]
image
The list and link styling is not standard, I have added specific styling for this page in the head of the release report to render ul > li circle style bullet points, and blue links.

File: templates/admin/release_report_detail.html [hunk 1]
```diff
@@ -8,6 +8,21 @@
...
+  }
+
+  a {
+    color: rgb(2, 132, 199);
+  }
+
+  a:hover {
+    color: rgb(255, 159, 0);
+  }
+}
```
Comment (brianjp93) [inline templates/admin/release_report_detail.html:25]: any styling for the sponsor message will need to be added here, or perhaps directly in the html in the `sponsor_message` in the Version model

File: templates/admin/release_report_detail.html [hunk 2]
```diff
@@ -8,6 +8,20 @@
...
+  margin-left: 2rem;
+
+}
+.sponsor-message a {
+  color: rgb(2, 132, 199);
+}
+
+.sponsor-message a:hover {
+  color: rgb(255, 159, 0);
+}
```
Comment (brianjp93) [inline templates/admin/release_report_detail.html:23]: any styling for the sponsor message will need to be added here, or perhaps directly in the html in the `sponsor_message` in the Version model

**Review by rbbeeston — APPROVED** [review]
LGTM!

**Comment by brianjp93** [comment]
@rbbeeston once deployed, just replace the sponsor_message in the admin with the HTML version of it and regenerate the report to see the styling applied.

**Pull request merged**